src/lib/things/Thing.svelte

```diff
@@ -36,7 +42,11 @@
     <div class="p-2">
         <img src={thing.image ?? BoxIcon} alt={thing.name} class="w-full aspect-square object-contain rounded" />
         <div class="mt-3">
-            <div class="{fontSize} uppercase font-bold font-sans text-center">{getShortName()}</div>
+            {#if language === "en"}
+                <div class="{fontSize} uppercase font-bold font-sans text-center">{getShortName(thing.name)}</div>
```

Comment:
Since the element we wrap the text in doesn't change, I think it would be cleaner to do something like this:
```svelte
<div class="{fontSize} uppercase font-bold font-sans text-center">
  {getShortName(language === 'en' ? thing.name : thing.spanishName)}
</div>
```

This way we're only changing which text gets inserted.